src/compas_fab/robots/planning_scene.py

```diff
@@ -59,6 +74,19 @@ def scale(self, transformation):
 class AttachedCollisionMesh(object):
     """Represents a collision mesh that is attached to a :class:`Robot`'s :class:`~compas.robots.Link`.
 
+    Parameters
+    ----------
+    collision_mesh : :class:`compas_fab.robots.CollisionMesh`
+        The collision mesh we want to attach.
+    link_name : :obj:`str`
+        The name of the :class:`~compas.robots.Link` the collision mesh will be
+        attached to.
+    touch_links : :obj:`list` of :obj:`str`
+        The list of link names the collision mesh is allowed to touch. Defaults
+        to the link it is attached to.
+    weight : :obj:`float`
+        The weight of the attached object. Defaults to ``1.0``.
```

Comment:
What do these weight do? I think it's unclear from a user side if you read this.

Reply:
It comes from weight as defined here: https://docs.ros.org/kinetic/api/moveit_msgs/html/msg/AttachedCollisionObject.html right?

Would it help to add that it's in kg? (Assumption based on [REP 103](https://www.ros.org/reps/rep-0103.html#units)

Reply:
ah! yes, i think it would. I was initially wondering if this would be a weight as in "weighted-average"

Reply:
That was my initial thought as well, but your question prompted me to dig deeper :).